users/README.md

```diff
@@ -2,26 +2,84 @@
 
 ```
 {
-  'id': string,
-  'username': string,
-  'first_name': string,
-  'last_name': string,
-  'email': string,
-  'phone': number,
-  'yoe': number,
-  'company': string,
-  'designation': string,
-  'img': string,
-  'github_id': string,
-  'linkedin_id': string,
-  'twitter_id': string,
-  'instagram_id': string,
-  'skills': [],
-  'website': string,
-  'github_display_name': string,
-  'isMember': boolean,
-  'userType': string,
-  'tokens': {},
-  'status': string
+  "id": <string>,
+  "username": <string>,
+  "first_name": <string>,
+  "last_name": <string>,
+  "email": <string>,
+  "phone": <number>,
+  "yoe": <number>,
+  "company": <string>,
+  "designation": <string>,
+  "img": <string>,
+  "github_id": <string>,
+  "github_display_name": <string>,
+  "linkedin_id": <string>,
+  "twitter_id": <string>,
+  "instagram_id": <string>,
+  "skills": [],
+  "website": <string>,
+  "isMember": <boolean>,
+  "userType": <string>,
+  "tokens": {
+    "githubAccessToken": <string>
+  },
+  "status": <string>,
+  "roles": {
+    "app_owner": <boolean>,
+    "archived": <boolean>,
+    "member" : <boolean>,
+    "restricted": <boolean>,
+    "super_user": <boolean>,
+    "inDiscord" : <boolean>,
+  },
+  "profileURL": <string>,
+  "picture":{
+    "publicId": <string>,
+    "url": <string>
+  }
 }
-```
\ No newline at end of file
+```
+
+### Example
+
+```
+{
+  "id": "12345",
```

Comment:
```suggestion
```json
{
  "id": "12345",
```
```suggestion
```json
{
```
Please use JSON code block

Reply:
I do not see the updated change of adding `json` in example section. Please check